newIDE/app/src/MainFrame/Editors/EventsFunctionsExtensionEditor.js

```diff
@@ -85,6 +85,12 @@ export default class EventsFunctionsExtensionEditorWrapper extends BaseEditor {
           initiallyFocusedBehaviorName={this.props.initiallyFocusedBehaviorName}
           onBehaviorEdited={this._onBehaviorEdited}
           ref={editor => (this.editor = editor)}
+          previewButtonSettings={{
```

Comment:
nitpicking: I just add an idea to make this a bit clearer and less cumbersome to use.

Export an object which is this "empty" settings, and then use it here:

* In PreviewButtons file: `export const emptyPreviewButtonSettings = { isPreviewOverride: false, ... }`
* `import { ..., emptyPreviewButtonSettings, ... } from '...'` (the PreviewButtons file)
* `previewButtonSettings={emptyPreviewButtonSettings}`